tendrl/node_agent/monitoring/collectd/collectors/gluster/low_weight/tendrl_glusterfs_health_counters.py

```diff
@@ -19,6 +19,7 @@ class TendrlGlusterfsHealthCounters(
 
     def __init__(self):
         self.provisioner_only_plugin = False
+        self.brick_path_repl = ":"
```

Comment:
We should not keep defining this in all the sources one by one. Rather define this as part of some configuration and use or at least in one common source and refer it.